CASMINST-4866 Layout Changes for install/README.md (#1934)
Cray-HPE/docs-csm: install/troubleshooting_ceph_csi.md

If there has been a failure to initialize all Ceph CSI components on `ncn-s001`, then the storage node
`cloud-init` may need to be rerun.

## Topics

1. [Verify Ceph CSI](#1-verify-ceph-csi)
1. [Rerun Storage Node `cloud-init`](#2-rerun-storage-node-cloud-init)

### 1. Verify Ceph CSI

Verify that the `ceph-csi` requirements are in place.

   1. (`ncn-s001#`) Log in to `ncn-s001` and run the following command.

      ```bash
      ceph -s
      ```

>       If it returns a connection error, then assume Ceph is not installed. See [Rerun Storage Node `cloud-init`](#2-rerun-storage-node-cloud-init).

Reviewer: ```suggestion
      If it returns a connection error, then assume Ceph is not installed. See [Rerun storage node `cloud-init`](#2-rerun-storage-node-cloud-init).
```

If there has been a failure to initialize all Ceph CSI components on `ncn-s001`, then the storage node
`cloud-init` may need to be rerun.

## Topics

1. [Verify Ceph CSI](#1-verify-ceph-csi)

> 1. [Rerun Storage Node `cloud-init`](#2-rerun-storage-node-cloud-init)

Reviewer: ```suggestion
1. [Rerun storage node `cloud-init`](#2-rerun-storage-node-cloud-init)
```

>       Check your results against this example.

Reviewer: ```suggestion
      Check the results against this example.
```

>       If any components are missing, see [Rerun Storage Node `cloud-init`](#2-rerun-storage-node-cloud-init).

Reviewer: ```suggestion
      If any components are missing, see [Rerun storage node `cloud-init`](#2-rerun-storage-node-cloud-init).
```

       ```bash
       ls /etc/cray/ceph
       ```

>        If any files are there they will represent completed stages.

Reviewer: ```suggestion
       If any files are there, they represent completed stages.
```

       ceph-csi configuration has been already been completed
       ```

>        - If your output is like above, then that means that all the steps ran.

Reviewer: ```suggestion
       - If the output is like above, then that means that all the steps ran.
```

>          - You can also locate the files in `/etc/cray/ceph` that are created as each step completes.

Reviewer: ```suggestion
         - Files in `/etc/cray/ceph` are created as each step completes and can be checked to confirm that the steps were run.
```
